scripts/version.js

```diff
@@ -76,59 +68,34 @@ async function run() {
 
     if (answer === false) return 0;
```

Comment:
If it's true we only use this for experimentals now, we might be able to shed some more stuff in here:

* This prompt stuff (step 2) isn't necessary since we're using `--skip-prompt`, so we could remove that CLI flag and this code
* The manual version bump above (`if (version == null)`) I think is no longer needed since we'll _always_ provide a version as a CLI arg?
* I think all the `isSnapshotVersion` conditionals could go away since it'll only be used for those

Reply:
Good call. I've simplified the script now so that the version specifier is required.

In the current package structure, the `isSnapshotVersion` is still required because it's used to detect experimental/nightly releases and ensure that `@remix-run/router` also gets a version update.